build.gradle

```diff
@@ -70,6 +70,7 @@ gradle.projectsEvaluated {
 dependencies {
     compile 'javax.validation:validation-api:1.1.0.Final'
     compile 'com.graphql-java:graphql-java:7.0'
+    compile 'org.clapper:javautil:3.2.0'
```

Comment:
This dependency is not a valid osgi bundle and is not correctly parsed by the osgi plugin, this can be an when deploying in a container. At least it would be good to have the dependency optional for osgi, but I don't really know how to do that with gradle

Reply:
So what can I do? I need this dependency to be able to search the class path

Reply:
hi, look at this one :[reflections]( https://github.com/ronmamo/reflections)